getting-started/web-app.md

```diff
@@ -20,10 +20,30 @@ This article explains how to use the Telerik UI for Blazor components into a <a
 
 1. Select the [**Blazor Web App**](https://learn.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-8.0#blazor-web-app) project type, enter a name for your project, and then click **Next**.
 
-1. Select the .NET 8 framework, the desired [interactive render mode](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes) and interactivity location (per page or global).
+1. Select the desired [Interactive render mode](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes) and Interactivity location (*Global* or *Per page / component*). See the [notes below](#interactivity-considerations) while choosing Interactivity location.
 
 1. Click **Create**.
 
+### Interactivity Considerations
```

Comment:
* My suggestion is to move this section to a new article. As this section contains important conceptual information, I don't recommend using a KB. Unfortunately, I don't find a section other than Getting Started that could accommodate this type of article. This makes me think that we should consider restructuring the current Getting Started section because it is starting to get too big.
* Once you move this content to a new article, make sure to update step 3 in https://docs.telerik.com/blazor-ui/getting-started/web-app#step-1-create-a-new-project